README.md

```diff
@@ -1051,6 +1052,12 @@ This list results from Pull Requests, reviews, ideas, and work done by 1100+ peo
 
 **[⬆️ Back to Top](#table-of-contents)**
 
+## Generative AI
+
+* [Portkey](https://portkey.ai/) - Control panel for Gen AI apps featuring an observability suite & an AI gateway. Send & log up to 10,000 requests for free every month.
```

Comment:
Thanks, we dont add single entry categories.

Please move other AI things in here or add your entry to an existing category.

Reply:
@ripienaar added a couple more tools in the category! Would that work?

Reply:
yes, unfortunately gradient doesnt list pricing nor do they specifically say the standard plan is free so I cant add it

Reply:
Oh shoot you're right. Sorry about that. Removed that, and added another tool. Hope that works.